controlplane/kubeadm/internal/workload_cluster_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	yaml2 "sigs.k8s.io/yaml"
```

Comment:
```suggestion
	"sigs.k8s.io/yaml"
```

Let's alias the capi owned yaml package to `utilyaml` here (we already do so in other packages)

Reply:
Also changed in other files, which are a part of this PR.

Reply:
would not have been necessary but also does not hurt 👍